Fix some typos so Git Workflow reads better (#55)
rootstrap/tech-guides: git/README.md

### Staging branch (staging) **[Required]**

This branch has all the features ready to be tested by the Client.

If the project has CD configured, every time something is merged into this branch the code will be automatically deployed to the Staging environment for the Client to test.

* **Origin:** *master* (in the initial project set up this branch is created from master)
* **Destination:** -
* **Use case:** Integration of the work done by the development team which is already tested and approved by the QA team and ready for the Client to review.

### QA branch (qa) **[Optional]**

This branch has all the features ready to be tested by the QA team.

If the project has CD configured, every time something is merged into this branch the code will be automatically deployed to the QA environment.

* **Origin:** *master* (in the initial project set up this branch is created from master)
* **Destination:** *staging*
* **Use case:** Integration of the work done by the development team ready for QA team to test.

> **_Note_**: *If a feature merged into the QA branch is not ready to be promoted to the Staging branch, needed adjustments (removing the feature/disabling access to it/etc) should be made in the Develop branch, followed by promoting the code to the QA branch to continue the flow.*

Reviewer: ```suggestion
**_Note_**: *If a feature merged into the QA branch is not ready to be promoted to the Staging branch, then all needed changes (like: fixes, removing the feature, disabling access to it, etc) should be made in the Develop branch, followed by promoting those changes to the QA branch to continue the flow.*
```

If there is a change of any kind in the feature branches in a commit (*git amend*) or in the history (*git rebase*), a new *demo* branch can be created to reflect the change or it can just be merged in the existing branch.

> **_Note_**: *In spite of the fact that this branch's main purpose is showing the progress in a review, it is not advised to send a version from that branch. In these cases you should talk to project or company referents.*

Reviewer: `In spite of` sounds too formal and out of place
```suggestion
**_Note_**: *Despite the fact that this branch's main purpose is showing the progress in a review, it is not advised to send a version from that branch. In these cases you should talk to project or company referents.*
```
